unlock-app/src/unlockTypes.ts

```diff
@@ -210,6 +211,7 @@ export interface PaywallConfig {
   minRecipients?: number
   superfluid?: boolean
   hideSoldOut?: boolean
+  password?: boolean
```

Comment:
I would move it inside the the lock object (same for `captcha`, `recurringPayments`, `maxRecipients`,  `minRecipients`, `superfluid` ... etc)